src/providers/v2/subgraph-provider.ts

```diff
@@ -317,4 +314,58 @@ export class V2SubgraphProvider implements IV2SubgraphProvider {
 
     return poolsSanitized;
   }
+
+  // This method checks if a given pool is part of the Virtual Pair Base V2 Pool IDs.
+  private isVirtualPairBaseV2Pool(pool: RawV2SubgraphPool): boolean {
+    return (
+      this.chainId === ChainId.BASE &&
+      V2SubgraphProvider.VirtualPairBaseV2PoolIds.has(pool.id.toLowerCase())
+    );
+  }
+
+  // TODO: Remove. Temporary fix to allow VIRTUAL/* V2 top pools to be included in the list until the subgraph is re-indexed.
+  private static readonly VirtualPairBaseV2PoolIds: Set<string> = new Set([
+    // Pool ID for VIRTUAL / GAME
+    '0xd418dfe7670c21f682e041f34250c114db5d7789',
```

Comment:
do we want all these pools, or do we want instead to have just virtual/eth? or will the other one not be found at all anyway so we at least need these?

Reply:
Yes we need those. `VIRTUAL/ETH` already goes through filtering.But it seems that usually best path for `ETH/{VirtualEcosystemToken}` swap goes through VIRTUAL, e.g. `ETH/GAME` swap -> `ETH/VIRTUAL/GAME` so we need to allow top `{VirtualEcosystemToken}/VIRTUAL` pools until subgraph re-indexes which might take a while.

Reply:
got it, understood